### DISCUSSION TITLE
Cannot mark as Answered II

### DISCUSSION THREAD

**Discussion by whistyun** [community/community#5242] [Discussions]
about https://github.com/github/feedback/discussions/2903#discussioncomment-1220341...

"Mark as answer" seem to have broken and not responding.
Perhaps this problem is caused by "Got It!".
This button is embedded with the token, but as a result the tokens appear to be duplicated.

![](https://user-images.githubusercontent.com/13712028/130456591-77499e13-1f0d-4c53-9a5f-3082a95e9ab0.png)

**Comment by oriapp** [comment]
Same issue...

**Comment by whistyun** [comment]
related #5249 and #5158.
